files/standard.json

```diff
@@ -9,7 +9,21 @@
         "link": "files/webcast.mov"
       }
     ],
-    "people": 42,
-    "chests": 9
+    "people": 5,
+    "chests": [
+      {
+        "sections": 2,
+        "targetDuration": 120
+      },
+      {
+        "sections": 3,
+        "targetDuration": 220
```

Comment:
220? mins? thats a lot since most escape last an hour

Reply:
It is seconds, but maybe an explanation of the jsonfile could be useful.

Reply:
duration is in seconden

Reply:
Is minutes not better? Should the host "guess" how much seconds it takes? Seems too specific for this kinda thing

Reply:
How is it guessing? And I think seconds isn't too specific, for when you want to have the half of a minute.